packages/common/types/control.types.ts

```diff
@@ -53,6 +54,17 @@ export interface RadioOption {
   value: string;
 }
 
+export interface Dropdown extends Omit<ControlBase, "value"> {
+  type: "dropdown";
+  value?: string;
+  options: string[] | DropdownOption[];
+}
+
+export interface DropdownOption {
```

Comment:
Lets have a `ControlOption` instead that is used by both radio and dropdown :)

Reply:
Sure!